app/views/hyrax/my/_work_action_menu.html.erb

```diff
@@ -27,7 +27,7 @@
 
     <li role="menuitem" tabindex="-1">
       <%= display_trophy_link(current_user, document.id) do |text| %>
-        <i class='glyphicon glyphicon-star'></i> <%= text %>
+        <i class='glyphicon glyphicon-star'></i> <%= t("hyrax.dashboard.my.action.highlight") %>
```

Comment:
Ah good catch! I missed that it switched according to context... easy fix. Thanks!